src/main/java/com/google/cloud/dataproc/templates/jdbc/README.md

```diff
@@ -44,3 +44,60 @@ jdbctobq.write.mode=
 # Spark warehouse directory location
 jdbctobq.spark.sql.warehouse.dir=
 ```
+
+
+## 2. JDBC To GCS
+
+Note - Add dependency jar's specific to database in jars variable.
+
+Example: export JARS=gs://<bucket_name>/mysql-connector-java.jar
+
+General Execution:
+
+```
+GCP_PROJECT=<gcp-project-id> \
+REGION=<region>  \
+SUBNET=<subnet>   \
+GCS_STAGING_LOCATION=<gcs-staging-bucket-folder> \
+HISTORY_SERVER_CLUSTER=<history-server> \
+export JARS=<gcs_path_to_jar_files> \
+
+bin/start.sh \
+-- --template JDBCTOGCS \
+--templateProperty jdbctobq.bigquery.location=<bigquery destination> \
+--templateProperty jdbctobq.jdbc.url=<jdbc url> \
+--templateProperty  jdbctogcs.output.location=<gcs-ouput-location> \
+--templateProperty  jdbctogcs.output.format=<output-format> \
+--templateProperty  jdbctogcs.write.mode=<write-mode> \
+--templateProperty  jdbctogcs.jdbc.url=<jdbc-url> \
+--templateProperty  jdbctogcs.jdbc.driver.class.name=<jdbc-driver-class-name> \
+--templateProperty  jdbctogcs.jdbc.properties={"user":"usernamehere","password":"passwordhere"} \
+--templateProperty  jdbctogcs.input.table=<input-table> \
+--templateProperty  jdbctogcs.input.db=<input-db> \
+--templateProperty  jdbctogcs.sql=<input-sql> \
+--templateProperty  jdbctogcs.partition.col=<partition-col> \
+```
```

Comment:
We should add some requirements of usage somewhere.
Like database must be reachable on the provided 'subnet' etc.

How to use this for SQL Query vs Single table export.

Reply:
Added the network requirements in README.md of jdbc template.